spec/models/glider_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Glider, type: :model do
+  fixtures :all
+
+  describe '#name' do
+    context 'glider name' do # how to improve this?
```

Comment:
If you only have one test case, such as here, you can leave the `context` off completely. If you have multiple text cases, then the contexts should describe what the difference in the test setup is between the cases.

Reply:
👍